float64 tosec
uint32 time
uint32 count
uint16 id
uint8 len
uint16[] data
uint16 status
uint16 aa
uint16 bb
uint16 cc
uint16 dd
uint16 ee
uint16 ff
uint16 gg
uint16 hh
uint16 ii
uint16 jj
uint16 kk
uint16 ll
uint16 mm
uint16 nn
uint16 oo
Header header

